pf/tests/provider_nested_custom_types_test.go

```diff
@@ -0,0 +1,290 @@
+package tfbridgetests
```

Comment:
Is there a reason why this isn't in `pf/internal/schemashim/`, similar to `custom_object_type_test.go`? I would move it there for consistency.

Reply:
I'm all for consistency - must've gotten lost in the mess. 👍

Reply:
Oh I remember now. The test provider is unavailable in `pf/internal` That's why. :/ I had it there first.